app/src/main/java/net/squanchy/notification/NotificationInjector.kt

```diff
@@ -0,0 +1,16 @@
+package net.squanchy.notification
+
+import android.app.Service
+
+import net.squanchy.injection.ApplicationInjector
+import net.squanchy.injection.ServiceContextModule
+
+internal object NotificationInjector {
+
+    fun obtain(service: Service): NotificationComponent {
```

Comment:
You can do this as a top level function, see `net/squanchy/venue/VenueInfoComponent.kt`for reference